SpecFlowSolar/Features/CartTests.feature

```diff
@@ -0,0 +1,11 @@
+﻿Feature: CartTests
+@mytag
+Scenario: add and delete product and return to home
+	Given open home page in CartTests
+	When open solar cabels in menu in CartTests
+	And add to cart one product with index 1
```

Comment:
the same - "index 1" is not a business language "add first product to cart" and in the Steps you can call something AddProduct(1)